src/components/GatherDetail/GatherDetail.tsx

```diff
@@ -52,90 +65,211 @@ const GatherDetail = ({
         <TextContainer>
-          <Text size={18}>모집 기간</Text>
-          <Text>{`${createdAt}~${deadline}`}</Text>
+          <Text size={18} strong>
+            모집 기간
+          </Text>
+          {isAuthor ? (
+            <EditDeadlineContainer>
+              <Text>{`${createdAt.substring(0, 10)} ~ `}</Text>
+              <Input
+                type="text"
+                name="deadline"
+                onChange={handleChange}
+                value={editValue.deadline}
+                customStyle={{ width: "30%" }}
+              />
+            </EditDeadlineContainer>
+          ) : (
+            <PeriodText
+              createdDate={createdAt.substring(0, 10)}
+              deadLine={deadline.substring(0, 10)}
+            />
+          )}
         </TextContainer>
         <TextContainer>
-          <Text size={18}>모집 인원</Text>
+          <Text size={18} strong>
+            모집 인원
+          </Text>
           <Text>{`${applicantLimit}명`}</Text>
         </TextContainer>
         <TextContainer>
-          <Text size={18}>신청 인원</Text>
+          <Text size={18} strong>
+            신청 인원
+          </Text>
           <Text>{`${applicantCount}명`}</Text>
           {participants?.map((applicant) => {
             return (
-              <ProfileBox
-                key={applicant.name}
-                src={applicant.profileImgUrl}
-                alt="프로필"
-                name={applicant.name}
-                course={applicant.course}
-                generation={applicant.generation}
-              />
+              <ApplicantContainer key={applicant.name}>
+                <BsFillPersonFill />
+                <ProfileBox
+                  src={applicant.profileImgUrl}
+                  alt="프로필"
+                  name={applicant.name}
+                  course={applicant.course}
+                  generation={applicant.generation}
+                  fontSize={14}
+                />
+              </ApplicantContainer>
             );
           })}
         </TextContainer>
         <TextContainer>
-          <Text size={18}>상세 내용</Text>
-          <Text>{content}</Text>
+          <Text size={18} strong>
+            상세 내용
+          </Text>
+          {isAuthor ? (
+            <MarkdownEditorWrapper>
+              <MarkdownEditor
+                editorRef={editorRef}
+                setEditorText={(value: string) =>
+                  setEditValue({ ...editValue, content: value })
+                }
+                value={editValue.content || ""}
+              />
+            </MarkdownEditorWrapper>
+          ) : (
+            <MarkdownEditorWrapper>
+              <MarkdownEditor
+                isViewMode
+                editorRef={editorRef}
+                setEditorText={(value: string) =>
+                  setEditValue({ ...editValue, content: value })
+                }
+                value={content || ""}
+              />
+            </MarkdownEditorWrapper>
+          )}
```

Comment:
```suggestion
<MarkdownEditorWrapper>
  <MarkdownEditor
    isViewMode={isAuthor ? false : true}
    editorRef={editorRef}
    setEditorText={(value: string) =>
      setEditValue({ ...editValue, content: value })
    }
    value={isAuthor ? (editValue.content || "") : (content || "")}
  />
</MarkdownEditorWrapper>
```

이렇게도 축약 해 볼 수 있을 것 같아요!

Reply:
코드 수정했습니다 짧아져서 보기 좋네요ㅎㅎ